### PR TITLE
wails: use Layout

### PR DESCRIPTION
**Pull request by Thykof**
This is a refacto that use our Layout in wails, it could solve some UI bugs.

### CODE REVIEW

**Comment by Thykof** [comment]
this does not work in the failure screen: the icon is not centered.

**Review by Thykof — COMMENTED** [review]
I Approve

File: wails-frontend/src/pages/failure.tsx [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
 import { promptRequest } from '../events/events';
+import { Layout } from '../layouts/Layout/Layout';
 
 const Failure = () => {
   const locationState = useLocation().state as { req: promptRequest };
   const req = locationState.req;
 
   return (
-    <div className="bg-primary flex flex-col justify-center items-center h-screen w-full">
-      <div className="w-1/4  max-w-xs  min-w-fit flex flex-col justify-center items-center">
```
Comment (mazmassa) [inline wails-frontend/src/pages/failure.tsx:11]: 👏

File: wails-frontend/src/pages/importFile.tsx [hunk 2]
```diff
@@ -54,31 +55,21 @@ const ImportFile = () => {
...
+      <h1 className="mas-title pb-4">{req.Msg}</h1>
+      <p className={account ? accountStyleSuccess : accountStyleNormal}>
+        {baselineStr()}
+      </p>
+      <div className="flex flex-row gap-4 pb-4">
+        <Button variant={'secondary'} onClick={handleCancel}>
+          Cancel
+        </Button>
+        <div className="min-w-fit">
+          <Button onClick={handleApply}>{importStr()}</Button>
```
Comment (mazmassa) [inline wails-frontend/src/pages/importFile.tsx:68]:
```suggestion
          <Button onClick={handleApply}>{getImportLabel()}</Button>
```

**Review by mazmassa — APPROVED** [review]

**Pull request merged**